### PR TITLE
Move all cql-utils functions to cql command

### PR DESCRIPTION
**Pull request by laodouya**
cql command now add three subcommands:
rpc
generate
idminer

### CODE REVIEW

File: client/driver.go [hunk 1]
```diff
@@ -283,8 +283,13 @@ func Drop(dsn string) (txHash hash.Hash, err error) {
 		return
 	}
 
+	_ = cfg
```
Comment (auxten) [inline client/driver.go]: Useless

File: cmd/cql/internal/generate.go [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+		fmt.Printf("\"%s\" already exists. \nDo you want to delete it? (y or n, press Enter for default n):\n",
+			path)
+		t, err := reader.ReadString('\n')
+		t = strings.Trim(t, "\n")
+		if err != nil {
+			ConsoleLog.WithError(err).Error("unexpected error")
+			SetExitStatus(1)
+			Exit()
+		}
+		if strings.Compare(t, "y") == 0 || strings.Compare(t, "yes") == 0 {
```
Comment (leventeliu) [inline cmd/cql/internal/generate.go]: Can use `strings.Equal` instead.

**Review by leventeliu — APPROVED** [review]
LGTM.

**Review by auxten — APPROVED** [review]
LGTM

**Pull request merged**